pkgs/by-name/mo/monophony/package.nix

```diff
@@ -0,0 +1,78 @@
+{ lib
+, fetchFromGitLab
+, python310Packages
+, wrapGAppsHook4
+, gst_all_1
+, gobject-introspection
+, gtk4
+, yt-dlp
+, libadwaita
+, libsoup_3
+, glib-networking
+,
+}:
+let
+  # pydbus from mpris_server is constrained to python310
```

Comment:
pydbus has [recently been patched](https://github.com/NixOS/nixpkgs/pull/263669) to work with python 3.11 if you rebase it should work again with python 3.11

Reply:
PR #264023 updates nuitka to latest stable release and adds support for python 3.11

Reply:
Done